hack/ci-test-e2e.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+set -euo pipefail
+
+set -o monitor
+
+set -x
+
+generate_junit() {
+  cat  /tmp/test_out | go tool test2json -t > /tmp/test_out.json
+  gotestsum --raw-command --junitfile="${ARTIFACT_DIR}/junit.xml" --format=standard-verbose -- cat /tmp/test_out.json
```

Comment:
do we need to run [junitmerge](https://github.com/openshift/release/tree/master/tools/junitmerge) against the the currently generated junit output?